src/rockstor/storageadmin/templates/storageadmin/base.html

```diff
@@ -251,59 +251,43 @@ <h3>Error!</h3>
-      <div class="modal-dialog">
-    <div class="modal-content">
-      <div class="modal-header">
-            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h3 id="donate-modal-label">Thank you! for supporting Rockstor development</h3>
-      </div>
-      <div class="modal-body">
-        <div class="radio">
-          <label>
-        <input type="radio" name="contrib" id="contrib10" value="10" checked>$10
-          </label>
-        <div class="radio">
-          <label>
-        <input type="radio" name="contrib" class="contrib-other" id="contrib20" value="20">$20
-          </label>
-        </div>
-        <div class="radio">
-          <label>
-        <input type="radio" name="contrib" class="contrib-other" id="contrib30" value="30">$30
-          </label>
-        </div>
-        <div class="radio">
-          <label>
-        <input type="radio" name="contrib" class="contrib-other" id="contrib50" value="50">$50 (Free Rockstor USB drive)
-          </label>
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal"
+                  aria-label="Close"><span aria-hidden="true">&times;</span>
+          </button>
+          <h2 id="donate-modal-label">Please consider donating to help support
+            Rockstor's development</h2>
         </div>
-        <div class="radio">
-          <label>
-        <input type="radio" name="contrib" id="contrib-custom" value="custom">Other
-          </label>
-        </div>
-        &nbsp;<input class="span2" id="custom-amount" name="custom-amount" placeholder="Enter amount" style="display: none">
-        <br><br>
-        <a id="donateYes" class="btn btn-primary" title="Donate">Donate</a>
-      </div>
-    </div><!-- /.modal-content -->
-      </div><!-- /.modal-dialog -->
+        <div class="modal-body">
+          <h5 id="donate-modal-subtitle">Our Open Source development is
+            dependant upon update channel subscriptions and donations.</h5>
+          <form id="inner-contrib-form"
+                action="https://www.paypal.com/cgi-bin/webscr" method="post"
+                target="_blank">
+            <input type="hidden" name="cmd" value="_s-xclick"/>
+            <input type="hidden" name="hosted_button_id"
+                   value="P72VDTYULUZTC"/>
+            <input type="image"
+                   src="https://www.paypalobjects.com/en_GB/i/btn/btn_donate_SM.gif"
+                   border="0" name="paypal_donate_button"
+                   id="paypal_donate_button"
+                   title="PayPal - The safer, easier way to pay online!"
+                   alt="Donate with PayPal button"/>
+            <!--      <img alt="" border="0" src="https://www.paypal.com/en_GB/i/scr/pixel.gif"-->
+            <!--           width="1" height="1"/>-->
+          </form>
```

Comment:
Add descriptive text:
```suggestion
          </form>
              <em>Upon clicking the above button, you will be redirected to our donation handler PayPal.</em>
```